test/custom.test.js

```diff
@@ -40,3 +41,9 @@ it('works alongside global usage query', () => {
     var list = browserslist('> 10% in my stats, > 1%', { stats: usage });
     expect(list.length > 1).toBeTruthy();
 });
+
+it('take stats from browserslist-stats.json', () => {
+    var data = JSON.parse(fs.readFileSync(statsFile));
```

Comment:
You pass a stats by `stats` argument in this test case. It doesn’t test that you look up in directories.

Look how I test a config file.